platform/core/src/services/HangingProtocolService/lib/validator.js

```diff
@@ -14,6 +14,41 @@ validate.validators.doesNotEqual = function(value, options, key) {
   }
 };
 
+// Ignore case contains.
+// options testValue MUST be in lower case already, otherwise it won't match
```

Comment:
can't we just force lower case in the function?

Reply:
Rather not - these are called potentially a huge number of times, and lower case is somewhat expensive.